Server/src/routes/track.route.ts

```diff
@@ -221,17 +221,17 @@ export class TrackRoute {
 				// If we know that, convert it in the API format.
 				const pos: Position | undefined = geo_pos
 					? {
-							lat: GeoJSONUtils.getLatitude(geo_pos),
-							lng: GeoJSONUtils.getLongitude(geo_pos)
-					  }
+						lat: GeoJSONUtils.getLatitude(geo_pos),
+						lng: GeoJSONUtils.getLongitude(geo_pos)
+					}
 					: undefined
 				// Also acquire the percentage position. It might happen that a percentage position is known, while the position is not.
 				// This might not make much sense.
-				const percentagePosition = trackKm
+				const percentagePosition: number | undefined = trackKm !== null && trackKm !== undefined
```

Comment:
Why the strict-inequality (`!==`) check with null AND undefined here?
`trackKm !== null && trackKm !== undefined` can be simplified to `trackKm != null`, as `null` is loosely-equal to `undefined`, and nothing but `undefined` (and `null`)